### PR TITLE
Devi/serial number

### PR DESCRIPTION
**Pull request by v-dmudili**
Created a folder for SerialNumber app & added all the project files for windows 10 Iot Enterprise.

Following changes are done with respect to Windows iot core.

1) file modified : serialnumber.csproj
    Updated the Target Platform Version from 10.0.17763 to 10.0.19041

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Generate and installing an app packages. 
    Updated the Windows Platform from IoT Core to IoT Enterprise
   

3) file modified : Package.appxmanifest 
    Updated Display name : Windows iot core to Windows 10 IoT Enterprise Serial Number
    Updated Package name : Windows iot core to Windows 10 IoT Enterprise Serial Number
    Updated Package Display name : Windows iot core to Windows 10 IoT Enterprise Serial Number

### CODE REVIEW

**Comment by v-dmudili** [comment]
@rsameser,@twarwick - kindly review

File: SerialNumber/CS/serialnumber.csproj [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    <AssemblyName>serialnumber</AssemblyName>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.19041.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.15063.0</TargetPlatformMinVersion>
+    <MinimumVisualStudioVersion>14</MinimumVisualStudioVersion>
+    <FileAlignment>512</FileAlignment>
+    <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <WindowsXamlEnableOverview>true</WindowsXamlEnableOverview>
+    <PackageCertificateKeyFile>serialnumber_StoreKey.pfx</PackageCertificateKeyFile>
```
Comment (neilsh-msft) [inline SerialNumber/CS/serialnumber.csproj:20]: The TargetPlatformMinVersion here is 10.0.15063.0.  In the README.md files the minimum version is listed as 17134.  Which number is correct?

Comment (v-dmudili) [reply SerialNumber/CS/serialnumber.csproj:20]: We updated Readme as The TargetPlatformMinVersion is 10.0.15063.0.

**Review by rsameser — APPROVED** [review]

File: SerialNumber/CS/serialnumber.csproj (hunk 1 above)
Comment (neilsh-msft) [reply SerialNumber/CS/serialnumber.csproj:20]: Looks like @rsameser changed it back to 17134.

**Pull request merged**